cirq-core/cirq/transformers/gauge_compiling/gauge_compiling_test_utils.py

```diff
@@ -73,6 +74,45 @@ def test_all_gauges(self, mock_select, seed):
             else:
                 _check_equivalent_with_error_message(c, nc, gauge)
 
+    def test_sweep(self):
```

Comment:
test utils code should be test ... see https://github.com/quantumlib/Cirq/blob/main/cirq-core/cirq/transformers/gauge_compiling/gauge_compiling_test_utils_test.py

Reply:
done.